src/components/profile-page/access-tokens/access-token-deletion-modal.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useCallback } from 'react'
+import type { CommonModalProps } from '../../common/modals/common-modal'
+import { CommonModal } from '../../common/modals/common-modal'
+import { cypressId } from '../../../utils/cypress-attribute'
+import { Button, Modal } from 'react-bootstrap'
+import { Trans, useTranslation } from 'react-i18next'
+import type { AccessToken } from '../../../api/tokens/types'
+import { deleteAccessToken } from '../../../api/tokens'
+import { dispatchUiNotification, showErrorNotification } from '../../../redux/ui-notifications/methods'
+
+export interface AccessTokenDeletionModalProps extends Pick<CommonModalProps, 'show' | 'onHide'> {
+  token: AccessToken
+}
+
+/**
+ * Modal that asks for confirmation when deleting an access token.
+ */
+export const AccessTokenDeletionModal: React.FC<AccessTokenDeletionModalProps> = ({ show, token, onHide }) => {
+  useTranslation()
+
+  const onConfirmDelete = useCallback(() => {
+    deleteAccessToken(token.keyId)
```

Comment:
We could add a spinner to this, while the request is being processed, but maybe this better kept in an issue for later…

Reply:
I created #1651 for this.